lib/src/catcher.rs

```diff
@@ -219,6 +219,8 @@ pub mod defaults {
                 teapot.", handle_418,
             421, "Misdirected Request", "The server cannot produce a response for this
                 request.", handle_421,
+            422, "Unprocessable Entity", "The server cannot process this request body.",
```

Comment:
Change this to: "The request was well-formed but was unable to be followed due to semantic errors."